org_status/org_status.py

```diff
@@ -137,4 +160,23 @@ def main():
 
         org_host = Host(token, org, verbose=args.verbose)
         org_status = aggregate_org_status(org_host, threads=args.threads)
+
+        if args.export_orgs:
+            export_data = encode_repo_list(org_status, args.format)
+            export_file = f'{getcwd()}/{args.export_orgs}'
```

Comment:
Filename might be absolute, please use https://docs.python.org/3.6/library/os.path.html to inflate instead.